project/Dependencies.scala

```diff
@@ -8,16 +8,18 @@
  */
 
 import sbt._
+import Keys._
 
 object Dependencies {
   // Keep in sync with .github CI build
   val Scala212 = "2.12.18"
   val Scala213 = "2.13.11"
-  val ScalaVersions = Seq(Scala212, Scala213)
+  val Scala3 = "3.3.0"
+  val ScalaVersions = Seq(Scala212, Scala213, Scala3)
 
   val PekkoVersion = "1.0.0-RC3"
 
-  val SlickVersion = "3.3.3"
+  val SlickVersion = "3.5.0-M4"
```

Comment:
one solution would be to use Slack 3.3.3 for Scala 2 and 3.5.0-M4 for Scala 3 - that way Scala 2 users won't have t use a pre-release milestone

Reply:
> one solution would be to use Slack 3.3.3 for Scala 2 and 3.5.0-M4 for Scala 3 - that way Scala 2 users won't have t use a pre-release milestone

There's 3.4.1. But I don't recommend having different versions. How soon do you need to release?

Reply:
This has already been resolved in latest PR, we will release when 3.5.0 is ready

Reply:
Just checking: is it the merge of this PR that you will wait for until slick 3.5.0 is released or is it the release of the pekko-persistence-jdbc library that will wait for this? (I just wonder since I'm in a project where we are hoping for a stable release of pekko-persistence-jdbc for Scala 2 soon. 😁)

Reply:
I would also advocate waiting for a Slick 3.5.0 release. I wouldn't be against merging this PR in order to create snapshots for Scala 3 if we make a future agreement that the first release of pekko-persistence-jdbc will contain Slick 3.5.0, otherwise we would have to revert this PR.